config/kcp/apiexport.yaml

```diff
@@ -14,10 +14,10 @@ spec:
   - group: ""
```

Comment:
I guess this file should not be part of the PR?

Reply:
It's still part of the repo and `make manifests` generates it https://github.com/redhat-appstudio/application-api/blob/main/Makefile#L12 and I am required by the CI to properly run that target https://github.com/redhat-appstudio/application-api/blob/main/.github/workflows/pr.yaml#L95